about.html

```diff
@@ -2,406 +2,109 @@
-								class="speaker"
-							/>
-						</div>
-						<div class="info">
-							<h3>Kibrewossen Y</h3>
-							<p class="position"><em>CEO, Safari ICT Solution PLC</em></p>
-							<hr />
-							<p class="bio">
-								Lorem, ipsum dolor sit amet consectetur adipisicing elit.
-								Mollitia, id. Lorem ipsum dolor sit amet consectetur adipisicing
-								elit. Iure totam vero libero quisquam excepturi ipsa?
-							</p>
+				<p>
+					Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quasi, odit.
+				</p>
+				<div class="summits">
+					<div class="year Apr2020">
+						<div class="overlay">
+							<h3>2020</h3>
+							<p>ADF Global Summit Algeria</p>
 						</div>
 					</div>
-					<div class="speaker">
-						<div class="profile-picture">
-							<img
-								src="./assets/images/speaker1.jpg"
-								alt="Ms. Abeba"
-								class="speaker"
-							/>
-						</div>
-						<div class="info">
-							<h3>Abeba T</h3>
-							<p class="position"><em>Agricultural Development Minister</em></p>
-							<hr />
-							<p class="bio">
-								Lorem, ipsum dolor sit amet consectetur adipisicing elit.
-								Mollitia, id. Lorem ipsum dolor sit amet consectetur adipisicing
-								elit. Iure totam vero libero quisquam excepturi ipsa?
-							</p>
+
+					<div class="year Jul2018">
+						<div class="overlay">
+							<h3>2018</h3>
+							<p>ADF Global Summit Congo</p>
 						</div>
 					</div>
-					<button class="btn">
-						MORE<span><i class="fa-fa fa-angle-down"></i></span>
-					</button>
 				</div>
 			</section>
-		</div>
-		<section class="partners">
-			<h2>Partner</h2>
-			<hr />
-			<div class="partners-logo">
-				<div class="row">
-					<div class="col cgiar">
-						<img src="./assets/images/CGIAR.png" alt="" />
-					</div>
-					<div class="col">
-						<img src="./assets/images/UNitar.png" alt="" />
-					</div>
-					<div class="col">
-						<img src="./assets/images/Google_DeepMind_Logo.png" alt="" />
-					</div>
-				</div>
-				<div class="row">
-					<div class="col wri">
-						<img src="./assets/images/worrld-resource.png" alt="" />
-					</div>
-					<div class="col cgiar">
-						<img src="./assets/images/CGIAR.png" alt="" />
-					</div>
-					<div class="col"></div>
-				</div>
-			</div>
-		</section>
 
-		<footer>
-			<div class="ft-row">
-				<div>
-					<a href="index.html"><img src="./assets/images/logo.png" alt="" /></a>
+			<footer>
+				<div class="ft-row">
+					<div>
+						<a href="index.html"
+							><img src="./assets/images/logo.png" alt="ADF log" class="logo"
+						/></a>
+					</div>
+					<span></span>
+					<p>2022 African Development Forum</p>
 				</div>
-
-				<p>2022 African Development Forum</p>
-			</div>
-		</footer>
-
-		<script
-			src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/js/bootstrap.bundle.min.js"
-			integrity="sha384-OERcA2EqjJCMA+/3y+gxIOqMEjwtxJY7qPCqsdltbNJuaOe923+mo//f6V8Qbsw3"
-			crossorigin="anonymous"
-		></script>
+			</footer>
```

Comment:
- [ ] There are small differences in the footer of the mobile version, please 🙏🏽 correct this section on the home page and the about page to be similar to the original version.

![image](https://user-images.githubusercontent.com/58642949/197134263-28044c0c-a474-4cc3-abfd-52263928eac4.png)